src/org/sosy_lab/java_smt/delegate/logging/LoggingBasicProverEnvironment.java

```diff
@@ -120,6 +121,13 @@ public ImmutableMap<String, String> getStatistics() {
     return wrapped.getStatistics();
   }
 
+  @Override
+  public ProofNode getProof() throws SolverException, InterruptedException {
+    ProofNode p = wrapped.getProof();
+    logger.log(Level.FINE, "proof", p);
```

Comment:
This would print the entire proof into the logger. You only need to log special events or errors, so you can remove this.